README.md

```diff
@@ -1,44 +1,51 @@
-<img align="right" src="images/logo.svg" alt="SimpleX logo" height="90">
+<h1 align="center">
+  <img src="images/simplex-chat-logo.svg" alt="SimpleX logo">
 
-# SimpleX chat
+</h1>
 
-## Private, secure, decentralized
+<p align="center">
+  <img alt="GitHub all releases" src="https://img.shields.io/github/downloads/simplex-chat/simplex-chat/total">
+  <img alt="GitHub all releases" src="https://img.shields.io/github/v/release/simplex-chat/simplex-chat">
+  <img alt="Twitter Follow" src="https://img.shields.io/twitter/follow/SimpleXChat?style=social">
+</p>
 
-[![GitHub build](https://github.com/simplex-chat/simplex-chat/workflows/build/badge.svg)](https://github.com/simplex-chat/simplex-chat/actions?query=workflow%3Abuild)
```

Comment:
The new header looks cool!

Regarding badges, I think we should keep this one

Reply:
Good suggestion, I missed this one off. The Git badges don't render properly so I'd need to find the Shields equivalent to get this to render correctly